src/test/java/jimmy/mcgymmy/storage/JsonAdaptedFoodTest.java

```diff
@@ -26,19 +26,19 @@
     private static final String INVALID_DATE = "20/04/20";
     private static final String INVALID_TAG = "##lucnsh";
 
-    private static final String VALID_NAME = TypicalFoods.NASI_LEMAK.getName().toString();
-    private static final String VALID_PROTEIN = Integer.toString(TypicalFoods.NASI_LEMAK.getProtein().getAmount());
-    private static final String VALID_FAT = Integer.toString(TypicalFoods.NASI_LEMAK.getFat().getAmount());
-    private static final String VALID_CARBS = Integer.toString(TypicalFoods.NASI_LEMAK.getCarbs().getAmount());
-    private static final String VALID_DATE = TypicalFoods.NASI_LEMAK.getDate().toString();
-    private static final List<JsonAdaptedTag> VALID_TAGS = TypicalFoods.NASI_LEMAK.getTags().stream()
+    private static final String VALID_NAME = TypicalFoods.getNasiLemak().getName().toString();
+    private static final String VALID_PROTEIN = Integer.toString(TypicalFoods.getNasiLemak().getProtein().getAmount());
+    private static final String VALID_FAT = Integer.toString(TypicalFoods.getNasiLemak().getFat().getAmount());
+    private static final String VALID_CARBS = Integer.toString(TypicalFoods.getNasiLemak().getCarbs().getAmount());
+    private static final String VALID_DATE = TypicalFoods.getNasiLemak().getDate().toString();
+    private static final List<JsonAdaptedTag> VALID_TAGS = TypicalFoods.getNasiLemak().getTags().stream()
```

Comment:
In this test class, `TypicalFoods` is imported and subsequently `TypicalFoods.getNasiLemak()` etc. are called to invoke the example foods. However in previous test classes (for example `FindCommandTest.java`), each individual command is imported i.e. `import static jimmy.mcgymmy.testutil.TypicalFoods.getNasiLemak;`. 

Though not really a big issue, maybe standardise to the latter method, as is common with most of our code base.

Reply:
Very minor issue. I think we can put this in another PR if needed

Reply:
Sure, no problem.